common/include/pcl/common/time.h

```diff
@@ -155,8 +155,9 @@ namespace pcl
 
       /** \brief Constructor.
         *
-        * \param[i] window_size number of most recent events that are
-        * considered in frequency estimation (default: 30) */
+        * \param[in] window_size number of most recent events that are
+        * considered in frequency estimation (default: 30)
+        * \return Event frequency */
```

Comment:
Do you mean that _doxygen_ throws warnings for constructors without `\return`? Weird...